drive/client.py

```diff
@@ -132,21 +132,21 @@ def get_or_create_folder(self, folder_name: str, parent_id: Optional[str] = None
 
         return self.create_folder(folder_name, parent_id)
 
-    def remove_file(self, file_id: str):
+    def remove_file(self, file_id: str) -> Literal[""]:
```

Comment:
This is accurate, but seems a little weird! Maybe you want to make this return a bool in the future?